apps/cli/providers/aws.py

```diff
@@ -14,12 +14,12 @@ def __init__(self) -> None:
         self.client = boto3.client("ec2")
 
     def regions_list(self):
-        return [region["RegionName"] for region in EC2.describe_regions()["Regions"]]
+        return [region["RegionName"] for region self.client.describe_regions()["Regions"]]
```

Comment:
there is a missing `in` here
```suggestion
        return [region["RegionName"] for region in self.client.describe_regions()["Regions"]]
```